grammars/JSONC.arm.tmLanguage.json

```diff
@@ -17,27 +17,27 @@
             "begin": "\\[",
             "beginCaptures": {
                 "0": {
-                    "name": "punctuation.definition.array.begin.json.arm"
+                    "name": "punctuation.definition.array.begin.json.arm-template"
                 }
             },
             "end": "\\]",
             "endCaptures": {
                 "0": {
-                    "name": "punctuation.definition.array.end.json.arm"
+                    "name": "punctuation.definition.array.end.json.arm-template"
```

Comment:
We are still keeping arm-template instead of azrm-template?

Reply:
That was the decision.  @alex-frankel Any issues with that?